scripts/utils/kakarot.py

```diff
@@ -228,6 +228,9 @@ async def deploy_and_fund_evm_address(evm_address: str, amount: float):
     """
     Deploy an EOA linked to the given EVM address and fund it with amount ETH
     """
+
```

Comment:
We are funding the account before, so that it has eth to pay back to the deployer.